Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"DesignSurface\">\r\n    <option name=\"filePathToZoomLevelMap\">\r\n      <map>\r\n        <entry key=\"..\\:/study/dogs-immune/jetPack_b_station/app/src/main/res/layout/activity_main.xml\" value=\"0.22\" />\r\n      </map>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_11\" project-jdk-name=\"1.8\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/build/classes\" />\r\n  </component>\r\n  <component name=\"ProjectType\">\r\n    <option name=\"id\" value=\"Android\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision b53888a014a2b1e0adbdc8db1e8cb2cd46f15a24)
+++ b/.idea/misc.xml	(date 1655801530981)
@@ -4,6 +4,8 @@
     <option name="filePathToZoomLevelMap">
       <map>
         <entry key="..\:/study/dogs-immune/jetPack_b_station/app/src/main/res/layout/activity_main.xml" value="0.22" />
+        <entry key="..\:/study/jetpack-mvc-mvp/jetPack_b_station/app/src/main/res/layout/activity_flow_player.xml" value="0.35260416666666666" />
+        <entry key="..\:/study/jetpack-mvc-mvp/jetPack_b_station/app/src/main/res/layout/activity_player.xml" value="0.5" />
       </map>
     </option>
   </component>
